### PR TITLE
Add Ant and Apache poi-scratchpad new versions

### PR DESCRIPTION
**Pull request by ShanChathusanda93**
## Purpose
- Add new Ant core orbit bundle
- Add Apache poi-scratchpad bundle

## Goals
- Update the ant core and poi-scratchpad versions

### CODE REVIEW

File: ant/1.10.12.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<!--
+~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (somindatommy) [inline ant/1.10.12.wso2v1/pom.xml:2]: Wrong license header

File: ant/1.10.12.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <properties>
+        <ant.version>1.10.12</ant.version>
+    </properties>
+</project>
```
Comment (somindatommy) [inline ant/1.10.12.wso2v1/pom.xml:87]: Add a newline at the end of the file

File: poi-scratchpad/5.2.3.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,88 @@
+<!--
+ ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (somindatommy) [inline poi-scratchpad/5.2.3.wso2v1/pom.xml:2]: Why is the year 2018?

File: ant/1.10.12.wso2v1/pom.xml (hunk 1 above)
Comment (ShanChathusanda93) [reply ant/1.10.12.wso2v1/pom.xml]: Updated

File: poi-scratchpad/5.2.3.wso2v1/pom.xml (hunk 3 above)
Comment (ShanChathusanda93) [reply poi-scratchpad/5.2.3.wso2v1/pom.xml]: Updated

File: poi-scratchpad/5.2.3.wso2v1/pom.xml [hunk 4]
```diff
@@ -0,0 +1,77 @@
+<!--
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
```
Comment (sachiniWettasinghe) [inline poi-scratchpad/5.2.3.wso2v1/pom.xml:2]: Update to the new license header

File: ant/1.10.12.wso2v1/pom.xml [hunk 5]
```diff
@@ -0,0 +1,78 @@
+<!--
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (ShanChathusanda93) [reply ant/1.10.12.wso2v1/pom.xml]: Updated

File: poi-scratchpad/5.2.3.wso2v1/pom.xml (hunk 4 above)
Comment (ShanChathusanda93) [reply poi-scratchpad/5.2.3.wso2v1/pom.xml]: Updated

**Review by somindatommy — APPROVED** [review]

**Review by sachiniWettasinghe — APPROVED** [review]

**Pull request merged**